scheduler/models/tasks.py

```diff
@@ -59,13 +61,13 @@ class TaskORM(Base):
     created_at = Column(
         DateTime(timezone=True),
         nullable=False,
-        default=datetime.datetime.utcnow,
+        server_default=func.now(),
     )
     modified_at = Column(
         DateTime(timezone=True),
         nullable=False,
-        default=datetime.datetime.utcnow,
-        onupdate=datetime.datetime.utcnow,
+        server_default=func.now(),
```

Comment:
I think that setting server_default also needs a migration because the table definition needs to changed with the default?

Reply:
Thought as well, but didn't show up when I made one. Will check it some more